docs/contribute.rst

```diff
@@ -296,6 +297,27 @@ squash the changes into a single commit. Use multiple commits to
 group relevant code changes if the pull request is too large for a
 single commit.
 
+If the pull request addresses an existing issue, mention it using
+one of the automatically parsed formats so that it is linked to
```

Comment:
Perhaps a link to https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests#linking-a-pull-request-to-an-issue ?

Reply:
Ack, linked in e62c0008.